sql_metadb/derived_tables/finance_transaction_invoices.sql

```diff
@@ -51,6 +51,7 @@ FROM
     LEFT JOIN folio_finance.fund AS tf ON ft.tofundid = tf.id
     LEFT JOIN folio_finance.budget AS fb ON ft.fromfundid = fb.fundid AND ft.fiscalyearid = fb.fiscalyearid
     LEFT JOIN folio_organizations. organizations AS oo ON jsonb_extract_path_text(ii.jsonb, 'vendorId')::uuid = oo.id
+    
 WHERE (jsonb_extract_path_text(ft.jsonb, 'transactionType') = 'Pending payment'
```

Comment:
Be careful not to make accidental changes, in this case only a harmless blank line inserted into the query.